### ISSUE TITLE
Meteors and twinkles don't stop after navigating off homepage

### ISSUE THREAD

**Issue by BruceMcRooster** [markhorn-dev/astro-sphere#4]
When navigating to any page besides `index.md`, errors are thrown in the Javascript console because the intervals that show meteors and twinkle stars continues to try to do so, even when the element they are targeting no longer exists.

The exact error, at least for the meteors, reads as follows
```
MeteorShower.astro:30 Uncaught TypeError: Cannot read properties of undefined (reading 'append') at createMeteor (MeteorShower.astro:30:10)
```

Furthermore, the homepage will "stack" the number of meteors and twinkles every time it is reopened. You can see this by repeatedly clicking back and forth between the homepage and some other page on the site.

**Comment by BruceMcRooster** [comment]
As I mentioned in #5, the errors are a less complicated but more pressing issue than the stacking. Red in the console is a lot worse than a few more stars, and I couldn't find an easy way to address the more stars. 
I'll leave it to the project maintainer to decide whether fixing the errors (which I did in #5) is enough, or if cracking down on overly-starry backgrounds is a must.

**Comment by markhorn-dev** [comment]
This should be fixed in #5 as well. The Astro View Transition api (which is not being used in this project) was present in the document head, and the scripts were being carried over to other pages.
